buildlib/tools/coverity.sh

```diff
@@ -55,7 +54,6 @@ run_coverity() {
 		echo "ok 1 Coverity found no issues"
 		rm -rf $cov_build
 	fi
-	module unload tools/cov
```

Comment:
why removed?

Reply:
Not need , there are in container

Reply:
IMO it should be a generic script, if it loads - should unload